### ISSUE TITLE
Unable to save, if field settings contains a `pattern`

### ISSUE THREAD

**Issue by samtrion** [sveltia/sveltia-cms#112]
![image](https://github.com/sveltia/sveltia-cms/assets/3283596/81d60167-f711-415c-9f4c-b04914acb6c3)

It is not possible for me to save currently, although the regular expression is fulfilled. Other variants of the regular expression are also impossible to save.

Configuration:
```yml
fields:
  - label: Slug
    name: slug
    widget: string
    i18n: duplicate
    pattern: ["[a-z0-9]{3,}", "The slug should be at least 3 characters long and only contain lowercase letters and numbers."]
    hint: "The slug is used to generate the URL of the author page. It should be unique and not contain any special characters."
```

**Comment by kyoshino** [comment]
Checking...

**Comment by kyoshino** [comment]
Shipped [v0.12.3](https://github.com/sveltia/sveltia-cms/releases/tag/v0.12.3) to address this and other validation errors 🙏🏼
